pyproject.toml

```diff
@@ -59,10 +59,26 @@ exclude_lines = [
 ]
 
 [tool.pyright]
-include = ["examples", "hikari"]
+include = ["hikari"]
+exclude = ["**/__init__.py", "hikari/internal/enums.py", "hikari/internal/fast_protocol.py"]
```

Comment:
This excludes them from verify-types, which we dont want

Reply:
They generate stubs so it doesn't matter.